src/Settings.cpp

```diff
@@ -154,7 +160,13 @@ void validate()
 		}
 		exit(0);
 	}
-	if (!workload_and_args) {
+
+	if (no_workload_flag && !continuous_print_flag) {
+		std::cerr << "-n only works if continous output (-c) is enabled." << std::endl;
+		exit(1);
+	}
+	
+	if (!workload_and_args && !(no_workload_flag && continuous_print_flag)) {
```

Comment:
I think this error message needs to be improved. I needed quite some time to understand what this means. This should hint at the -h option at least.